docs/user/language-bindings/android/gecko-content-processes-and-glean.md

```diff
@@ -0,0 +1,16 @@
+## GeckoView Content Processes and Glean
+
+This is a brief overview of how GeckoView content processes record data in Glean, despite Glean being required to be called only from the main process.
```

Comment:
Wellllll.... It's more like "Only one Glean can access a single datareporting dir at a time, and since the datareporting dir is set for the whole app this means only one process can have an active Glean instance" aka the "One Glean to Rule Them All" convention

Reply:
We don't really go into that in other places in the docs, we only _require_ that Glean be initialized in the main process, otherwise special care must be taken (such as passing an alternative dataDir, ala VPN's daemon process), or like this, using some actual IPC. I feel like that might be getting a little too into the weeds for the expected audience of this doc.